src/chrome/content/rules/Xwiki.org.xml

```diff
@@ -0,0 +1,29 @@
+<!--
+	Plaintext response:
```

Comment:
```bash
$ curl https://lists.xwiki.org/mailman/listinfo
curl: (35) error:140770FC:SSL routines:SSL23_GET_SERVER_HELLO:unknown protoco
```

I get SSL errors

Reply:
curl fails to perform a SSL/TLS handshake. You can verify the plaintext response by forcing http on port 443: http://lists.xwiki.org:443/

Reply:
Yes. I can reproduce the plaintext response that SSL/TLS is not performed over port 443. thanks for explaining!